docs/starters.yml

```diff
@@ -5446,3 +5446,14 @@
     - Light and Dark themes based on CSS variables (persisted state)
     - Font Awesome
     - Normalize.css
+- url: https://gatsby-notion-demo.netlify.com/
+  repo: https://github.com/conradlin/gatsby-starter-strata-notion
+  description: Gatsby starter utilizing Notion as a CMS based on strata site template
+  tags:
+    - CMS: Notion
```

Comment:
Since Notion doesn't have an official API yet there aren't starters tagged as using it as a CMS (the list of valid tags is at this link: https://github.com/gatsbyjs/gatsby/blob/master/docs/categories.yml)

```suggestion
```

Can we remove this tag for now?

Reply:
No problem, thanks Kyle. Can you please suggest a suitable tag then? I simply want to help make this integration more known in the gatsby community, as it's been super useful for me!

Reply:
Looking in your `gatsby-config`, it looks like your starter will output a PWA with the offline and manifest plugins. You also have `react-helmet` which I assume would be for SEO, so perhaps the PWA and SEO tags? You're also using SCSS so maybe the Styling:SCSS tag?

As for a CMS tag, since there aren't any other existing starters using Notion and it isn't _technically_ officially supporting using it as a CMS yetm, I'm hesitant to add it as a tag now.

Reply:
Thanks for the feedback! I've committed an updated version with your suggested changes.